### PR TITLE
Explain horses

### PR DESCRIPTION
**Pull request by kripken**
For they may be puzzling.

### CODE REVIEW

**Comment by jfbastien** [comment]
I like.

File: BananaBread/bb.html [hunk 1]
```diff
@@ -1203,6 +1203,9 @@
       <progress value="0" max="100" id="progress" hidden=1></progress>
     </div>
 
+    <br>
```
Comment (jfbastien) [inline BananaBread/bb.html]: Isn't `<br />` what the cool kids do nowadays? Or is that so 2000?

Comment (kripken) [inline BananaBread/bb.html]: I think you're thinking of XHTML, which is probably from around 2000, yeah. In HTML you [shouldn't close the tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br).

**Comment by s3ththompson** [comment]
Closing given the focus on Angry Bots.

**Pull request closed without merging**